### DISCUSSION TITLE
Can't find MacOS Hardrive on boot screen

### DISCUSSION THREAD

**Discussion by VincentShin** [osy86/HaC-Mini#922] [General]
After I setup HaC-Mini from 3.0.1 to 3.1.0 , I can't find my MacOS HD on OC boot screen . Then I try to use the USB flash disk Hac-mini，To boot my MacOS , Failed. Finally , I replace the hardrive to another Macbookpro , and reinstall the 3.0.1 on this HD . my system return to normal now.

**Comment by VincentShin** [comment]
I forget said this , my system version is Catalina 10.15.7 . And the device is Hades canyon 8705G

**Reply by osy** [reply]
Can you open an issue and fill in the info?

**Reply by VincentShin** [reply]
Considering this device is my workhorse，Maybe I try to reproduce the issue on weekend.

**Reply by osy** [reply]
Are you on NVMe or SATA SSD?

**Comment by VincentShin** [comment]
Sorry for miss this message，
I’m using a NVMe disk: HIKVISION c2000pro.
It’s still not working on hacmini3.1.0 or later version.
